region/brinstar/kraid.json

```diff
@@ -536,7 +536,19 @@
                     {"enemyDamage": {
                       "enemy": "Beetom",
                       "type": "contact",
-                      "hits": 5
+                      "hits": 4
+                    }}
+                  ]
+                },
+                {
+                  "name": "Tricky Evade",
```

Comment:
I imagine there is a particular way to evade and I'd like that as a note.

Reply:
Looks like you wait by the door briefly for 7 hit.
Using the speedrun item grab I could get 5 hit but that takes run speed entering